profile/pyroscope.go

```diff
@@ -0,0 +1,63 @@
+package profile
+
+import (
+	"github.com/pkg/errors"
+	"github.com/pyroscope-io/client/pyroscope"
+	"github.com/skit-ai/vcore/env"
+)
+
+var (
+	host, appName, releaseVersion string
+)
+
+func init() {
+	host = env.String("PYROSCOPE_HOST", "")
+	appName = env.String("APP_NAME", "")
+	releaseVersion = env.String("RELEASE_VERSION", "")
+}
+
+var (
+	errNoHost = errors.New("host not defined")
+)
+
+// InitPyroscope initialises profiling using Pyroscope.
+func InitPyroscope() error {
+	if host == "" {
+		return errNoHost
+	}
+
+	// Initialize Pyroscope with default profileTypes
+	err := initPyroscope(nil)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// InitPyroScopeWithProfiles initialises profiling wit specified ProfileTypes.
+// List of profiles can be found here: https://github.com/grafana/pyroscope-golang/blob/main/pyroscope/types.go
+func InitPyroScopeWithProfiles(profileTypes []pyroscope.ProfileType) error {
```

Comment:
I think it should be `InitPyroscopeWithProfiles` as mentioned in the readme.

Reply:
This is fixed.